wnfs/src/private/directory.rs

```diff
@@ -616,6 +616,94 @@ impl PrivateDirectory {
         }
     }
 
+    /// Opens a mutable reference to the specified file.
+    /// If the file is missing, it initializes an empty file and give a mut reference to that.
+    /// If the file already exists, it will copy it to the next revision, update the edit time, and give a mut reference to that.
+    /// # Examples
+    /// ```
+    /// use std::rc::Rc;
+    /// use chrono::Utc;
+    /// use rand::thread_rng;
+    /// use wnfs::{
+    ///    private::{PrivateForest, PrivateRef, PrivateDirectory},
+    ///    common::{BlockStore, MemoryBlockStore},
+    ///    namefilter::Namefilter,
+    /// };
+    /// #[async_std::main]
+    /// async fn main() {
+    ///    let store = &mut MemoryBlockStore::default();
+    ///    let rng = &mut thread_rng();
+    ///    let mut forest = &mut Rc::new(PrivateForest::new());
```

Comment:
This doesn't need to be `mut`, it's just a constant reference to something that's mutable:
```suggestion
    ///    let forest = &mut Rc::new(PrivateForest::new());
```